src/eholdings.test.js

```diff
@@ -0,0 +1,95 @@
+import {
+  cleanup,
+  render,
+} from '@testing-library/react';
+
+import { createMemoryHistory } from 'history';
+import Harness from '../test/jest/helpers/harness';
+
+import EHoldings from './eholdings';
+
+jest.mock('impagination/src/state', () => () => (<div>State component</div>));
+jest.mock('./routes/provider-show', () => () => (<div>ProviderShow component</div>));
+
+const location = {
+  hash: '',
+  key: '',
+  pathname: '',
+  search: '',
+  state: {},
+};
+
+const match = {
+  isExact: true,
+  params: {},
+  path: '',
+  url: '',
+};
+
+const root = {
+  addEpic: jest.fn(),
+  addReducer: jest.fn(),
+};
+
+const testProps = {
+  root,
+  location,
+  match,
+  showSettings: false,
+};
+
+describe('Given EHoldings', () => {
+  const renderEHoldings = ({ history, pathLocation, pathMatch, ...props }) => render(
+    <Harness history={history}>
+      <EHoldings
+        root={root}
+        history={history}
+        location={pathLocation}
+        match={pathMatch}
+        {...props}
+      />
+    </Harness>
+  );
+
+  afterEach(cleanup);
+  let history = null;
+
+  beforeEach(() => {
+    history = createMemoryHistory();
+  });
+
+  it('should render navigation panel', () => {
```

Comment:
Why do we need this tests if they are commented?

Reply:
There was problems while testing this component so temporarily tests were commented. Now a separate ticket created to test EHoldings, so this test file is now removed.